app/api/api_extension.py

```diff
@@ -8,7 +8,10 @@
     + "through 1:1 relations during a certain period of time. \n \n"
     + "The main repository of the Backend System can be found here: https://github.com/anitab-org/mentorship-backend \n \n"
     + "The Android client for the Mentorship System can be found here: https://github.com/anitab-org/mentorship-android \n \n"
-    + "For more information about the project here's a link to our wiki guide: https://github.com/anitab-org/mentorship-backend/wiki"
+    + "For more information about the project here's a link to our wiki guide: https://github.com/anitab-org/mentorship-backend/wiki \n \n"
+    + "<b>Quality Assurance Tables:</b>  Quality Assurance Tables are table containing Test Case and its Outcome as <i>Sucess</i> or <i>Fail</i>, the intention is to tell the contributors about test cases for each feature implemented on the backend. \n \n"
+    + "The link to the Quality Assurance Test cases found <a href=https://github.com/anitab-org/mentorship-backend/blob/develop/docs/quality-assurance-test-cases.md>here</a>."
```

Comment:
```suggestion
    + "This <a href=https://github.com/anitab-org/mentorship-backend/blob/develop/docs/quality-assurance-test-cases.md>Quality Assurance Test cases</a> document contains examples of test scenarios evaluate if the API is working as it should."
```
I am not sure but line 12 is not showing up on your PR image 🤔 so I would frame it that way. What do you think? If you use this suggestion please delete line 12 :)

Reply:
Sure, let me fix it and update my PR.

Reply:
@isabelcosta I forget and wrote commit message as same as issue(add link to quality assurance tables to Swagger UI) title is should be like this (added link to quality assurance tables to Swagger UI) is that all right?

Reply:
It's ok :) Just keep in mind the Commit Message Guidelines you can find on our contribution guidelines.